dev/ci/ci-elpi.sh

```diff
@@ -10,7 +10,7 @@ git_download elpi
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/elpi"
-  make build-core
-  make build-apps
-  make install
+  make build-core DUNE_build_FLAGS="--root ."
+  make build-apps DUNE_build_FLAGS="--root ."
+  make install DUNE_install_FLAGS=--prefix="$CI_INSTALL_DIR"
```

Comment:
make install does
~~~
dune build -p coq-elpi
dune install coq-elpi
~~~
this causes double building of coq-elpi since the previous build was in non-release mode

Also it builds some examples I guess because they are in the coq-elpi package, but we might like it better if they were in the test job instead

Reply:
> this causes double building of coq-elpi since the previous build was in non-release mode

Frankly I never understood why it is done like that, maybe @rlepigre had a reason.
In my daily use, I don't think it is built twice when I install.

Reply:
> builds some examples I guess because they are in the coq-elpi package, but we might like it better if they were in the test

I know, I will eventually fix that upstream

Reply:
What is the relation of @rlepigre to this PR?

Reply:
He ported Coq-Elpi do dune and wrote the install target

Reply:
we call dune install by hand in the ci job if we need to, but your question is really for him

Reply:
fixed

Reply:
I don't remember any particular reason to set things up like that (I probably took inspiration from somewhere, but not sure where). Building twice was definitely not my intent.